editor/gui/scene_tree_editor.cpp

```diff
@@ -1531,6 +1548,8 @@ void SceneTreeDialog::set_valid_types(const Vector<StringName> &p_valid) {
 		label->set_text(type);
 		label->set_auto_translate(false);
 	}
+
+	show_all_nodes->set_visible(true);
```

Comment:
```suggestion
	show_all_nodes->show();
```
This is used more commonly if the argument is constant.

Reply:
Fixed.